### ISSUE TITLE
riff build command time out - Error context deadline exceeded

### ISSUE THREAD

**Issue by ericbottard** [projectriff/riff#311]
_From @jldec on February 21, 2018 1:58_

I got this doing `riff create` for square with no cached images, on slowish (2Mb) wifi.

```
riff/samples/node/square (master)$ riff create --force
Initializing /Users/jleschner/riff/riff/samples/node/square/square-topics.yaml
Initializing /Users/jleschner/riff/riff/samples/node/square/square-function.yaml
Initializing /Users/jleschner/riff/riff/samples/node/square/Dockerfile
Initializing /Users/jleschner/riff/riff/samples/node/square/.dockerignore
Building image ...
signal: killed: 
Command timed out
Error context deadline exceeded
context deadline exceeded
```

If possible, I think it would be nice to display the progress ouput from the `docker build` and perhaps remove the timeout entirely (or offer a way to override it).

Running the `docker build` manually took a while but it worked.

_Copied from original issue: projectriff/riff-cli#51_

**Comment by ericbottard** [comment]
_From @trisberg on February 21, 2018 2:13_

That looks like a docker timeout due to network issues, not sure there is much we can do in the CLI

**Comment by ericbottard** [comment]
_From @jldec on February 21, 2018 2:25_

aah, fair enough.
is there any way to relay the docker progress to stdout?

**Comment by ericbottard** [comment]
_From @trisberg on February 21, 2018 2:27_

not sure, will take a look to see what can be done

**Comment by ericbottard** [comment]
_From @jldec on February 21, 2018 23:22_

actually i don't believe this is a docker timeout

i keep getting the cli error, but when i invoke the `docker build... ` (using --dry-run to figure it out), the build succeeds.

**Comment by trisberg** [comment]
Fixed via #342
